Assets/Scripts/UI/UI_TileInfoManager.cs

```diff
@@ -0,0 +1,35 @@
+namespace CT6GAMAI
+{
+    using TMPro;
+    using UnityEngine;
+
+    public class UI_TileInfoManager : MonoBehaviour
+    {
+        [SerializeField] TMP_Text TerrainValueText;
```

Comment:
Private variables here aren't following naming conventions. They should also state they're private

Reply:
Can use [FormerlySerializedAs] -> https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html